release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1235,21 +1233,21 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
+* Previously, when updating SSH keys, the Machine Configuration Operator (MCO) changed the owner and group of the `authorized_keys` file to `root`. This update ensures that the MCO preserves `core` as the owner and group when it updates the `authorized_keys` file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1956739[BZ#1956739])
 
 * Previously, a warning message sent by the `clone_slave_connection` function was incorrectly stored in a `new_uuid` variable, which is intended to store only the connection's UUID. As a result, `nmcli` commands that include the `new_uuid` variable were failing due to the incorrect value being stored in the `new_uuid` variable. With this fix, the `clone_slave_connection` function warning message is redirected to `stderr`. Now, `nmcli` commands that reference the `new_uuid` variable do not fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022646[*BZ#2022646*])
 
 * Previously, an old version of the Kubernetes client library was present in the `baremetal-runtimecfg` project. When a Virtual IP (VIP) failed, the client connections might not be closed in a timely manner. This could result in long delays for monitor containers that rely on talking to the API. This fix updates the client library. Now, connections are closed as expected on VIP failovers and the monitor containers do not hang for an excessive period of time. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1995021[*BZ#1995021*])
 
-* Before this update, the Machine Config Operator (MCO) stored pending configuration changes to disk before it applied them to {op-system-first}. If a power loss interrupted the MCO from applying the configuration, it treated the configuration as applied and did not validate the changes. If this configuration contained invalid changes, applying them failed. With this update, the MCO saves a configuration to disk only after being applied. This way, if the power is lost while the MCO is applying the configuration, it reapplies the configuration after it restarts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916169[*BZ#1916169*])
+* Before this update, the Machine Configuration Operator (MCO) stored pending configuration changes to disk before it applied them to {op-system-first}. If a power loss interrupted the MCO from applying the configuration, it treated the configuration as applied and did not validate the changes. If this configuration contained invalid changes, applying them failed. With this update, the MCO saves a configuration to disk only after being applied. This way, if the power is lost while the MCO is applying the configuration, it reapplies the configuration after it restarts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916169[*BZ#1916169*])
```

Comment:
The previous version was consistent with exiting usage - I think this is in the same boat as Ignition config. (This is true for all the MCO and MCD instances.)